modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc

```diff
@@ -103,4 +103,47 @@ An example could look like this:
 DROP DATABASE customers IF EXISTS DUMP DATA
 ----
 
+[[delete-databases-with-aliases]]
+=== Delete a database with local database aliases targeting it
+
+In general, deleting a database that is the target of any local database aliases will fail due as these aliases need to be dropped first.
+Remote database aliases targeting the database do not affect the deletion of the database.
+
+By appending `CASCADE ALIASES` to the command `DROP DATABASE`, those local database aliases can be dropped alongside the database.
+Any remote database aliases targeting the database will not be affected of the `CASCADE ALIASES`, they will simply no longer resolve their targets (which is the same behavior as if they are created targeting a non-existing database).
+However, the behavior does not cascade further, if any of the dropped database aliases are constituents to any composite database, those composite databases are not dropped.
+
+Using `CASCADE ALIASES` requires the `DROP ALIAS` privilege (see xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-alias-management[ALIAS MANAGEMENT privileges]).
+
+Given database and database alias:
```

Comment:
wouldn't this and the next bit make them into two separate examples? this is more set-up for the drop part and feels odd to have as an example on its own here?

Reply:
Actually, I would suggest to be one example within an example framework. But I need to build it locally to see how it will look. I find it difficult to see the result from just commenting.

Reply:
I'll suggest something.

Reply:
We can have a similar example and wording for the other example. But I think we should see if we can reorganize that page as well, as at the moment, it discusses two topics—one is in the intro (_You can delete composite databases using either the command DROP COMPOSITE DATABASE name or the more general one DROP DATABASE name_), and the other is the new section, which is another way of dropping a composite database.